src/server_manager/cloud/gcp_api.ts

```diff
@@ -119,7 +129,7 @@ export class RestApiClient {
    */
   createInstance(
       projectId: string, name: string, zoneId: string, size: string,
-      installScript: string): Promise<Operation> {
+      installScript: string): Promise<ComputeEngineOperation> {
     const data = {
```

Comment:
Pass data as a parameter. Move it to the caller.
The gcp_api should be unaware of Outline